cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -436,12 +437,12 @@ private void setOutputRootPath(File inputProject) {
                     LOG.info("(" + selectedArtifactNumber + ") " + artifactDescription);
                 } catch (IndexOutOfBoundsException e) {
                     LOG.error("The {} number you have specified is out of bounds!", artifactType);
-                    System.exit(1);
+                    throw (e);
```

Comment:
We should add a message for the thrown exception here too I think.

Reply:
We did. see the log above

Reply:
ok.